image-sync/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM python:3-slim AS build
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# hadolint ignore=DL3008
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+               build-essential gcc git \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /image-sync
+
+RUN python -m venv /image-sync/venv
+ENV PATH="/image-sync/venv/bin:$PATH"
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN pip install --no-cache-dir .
+
+# The actual image
+FROM python:3-slim
+
+LABEL org.opencontainers.image.source=https://github.com/EGI-Federation/fedcloud-catchall-operations
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# hadolint ignore=DL3015, DL3008
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       gnupg2 qemu-utils curl \
```

Comment:
Do we really need `qemu-utils` as dependency?

Reply:
yes, image conversion needs this

Reply:
ah, sorry, of course